.kitchen.yml

```diff
@@ -103,3 +103,8 @@ suites:
       name: terraform
       command_timeout: 1800
       root_module_directory: test/fixtures/mig_with_percent/simple
+  - name: mig_stateful
```

Comment:
Don't think we need this anymore since you have a go test for this now.

Reply:
Removing it